org.openhab.binding.zigbee/README.md

```diff
@@ -219,18 +238,20 @@ The following devices have been tested by openHAB users with the binding. The ab
 | Tradfri Repeater                               |                                                       |
 | Trust Bulbs                                    | *[<sup>[1]</sup> ](#note1)*                           |
 | Ubisys modules                                 | D1 Dimmer, S1/S2 Switch modules                       |
-| Xiaomi Aqara Door and Window Sensor            |                                                       |
-| Xiaomi Aqara Temperature and Humidity Sensor   |                                                       |
-| Xiaomi Aqara Human Motion Sensor               |                                                       |
-| Xiaomi Aqara Wireless Mini Switch              |                                                       |
-| Xiaomi Aqara Wired Wall Switch                 |                                                       |
-| Xiaomi Aqara Wireless Remote Switch            | Double Rocker variant                                 |
+| Xiaomi Aqara Door and Window Sensor            | *[<sup>[3]</sup> ](#note3)*                           |
+| Xiaomi Aqara Temperature and Humidity Sensor   | *[<sup>[3]</sup> ](#note3)*                           |
+| Xiaomi Aqara Human Motion Sensor               | *[<sup>[3]</sup> ](#note3)*                           |
+| Xiaomi Aqara Wireless Mini Switch              | *[<sup>[3]</sup> ](#note3)*                           |
+| Xiaomi Aqara Wired Wall Switch                 | *[<sup>[3]</sup> ](#note3)*                           |
+| Xiaomi Aqara Wireless Remote Switch            | Double Rocker variant *[<sup>[3]</sup> ](#note3)*     |
 | Shenzhen Ksentry Electronics On Off Controller | Relay                                                 |
 
 <a name="note1"></a> *Note 1: Some devices may not work with the Telegesis dongle.*
 
 <a name="note2"></a> *Note 2: The Hue Dimmer can be integrated but needs additional rule-configuration to work properly. See below for example.*
 
+<a name="note3"></a> *Note 3: Xiaomi devices are not fully ZigBee compliant, and don't play well with some routers. Please refer to [this}(https://community.hubitat.com/t/xiaomi-aqara-devices-pairing-keeping-them-connected/623) thread for more information.*
```

Comment:
I can sum it up, maybe in a separate section. I'd still like to keep the links somewhere as a reference, as getting to this information took me quite some googling - I'll make sure they're only listed as references, and not as the primary source for info.

Reply:
Ok, that sounds a good compromise. I just want to make sure the information doesn't disappear one day as it's something we can't control when it's just linked to another page...

Reply:
Any chance you are willing to enable the wiki features on this repo, and we can use that for this kind of documentation?

Keeping it in the README is just too static: products come and go, firmwares are updated. I was thinking about creating a dedicated thread on OpenHAB's discourse, but then I'll have to keep the pinned post up-to-date. A publicly-editable, gated page seems the best solution, can be linked from the README, and doesn't risk disappearing.

Reply:
No - I think that if we enable the wiki, we end up with information in multiple places - it's already in the forum, and also in the official documentation. Adding wiki to that will just add more confusion I think.

The README is publicly editable - it only requires a GitHub account which is the same as the wiki. If we add stuff into the README, then it is in the official documentation.

I have thought about doing some sort of database for ZigBee devices - similar to the ZWave database so that people can update this. There would be a lot of advantages to that as it can also directly flow into the documentation in a standard way, but it's another thing to do. I think for now at least the best option is the binding documentation.